File: backend/src/main/java/ropold/backend/model/MemoryModel.java
package ropold.backend.model;


public record MemoryModel(
        String id,
        String name,
        int matchId,
        Category category,
        String description,
        boolean isActive,
        String appUserGithubId,
        String appUserUsername,
        String appUserAvatarUrl,
        String appUserGithubUrl,
        String imageUrl
) {
}

File: frontend/src/components/model/MemoryModel.ts
import { Category } from "./Category.ts";

export type MemoryModel = {
    id: string;
    name: string;
    matchId: number;
    category: Category;
    description: string;
    isActive: boolean;
    appUserGithubId: string;
    appUserUsername: string;
    appUserAvatarUrl: string;
    appUserGithubUrl: string;
    imageUrl: string;
}


